src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IVsService`1.cs

```diff
@@ -15,7 +15,8 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS
     ///     The type of the service to retrieve and return from <see cref="GetValueAsync"/>.
     /// </typeparam>
     [ProjectSystemContract(ProjectSystemContractScope.Global, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
-    internal interface IVsService<T> where T : class
+    internal interface IVsService<T> 
+        where T : class?
```

Comment:
Is this documented anywhere? I searched around a bit but couldn't find anything.

Reply:
https://github.com/dotnet/docs/issues/17355